package.json

```diff
@@ -24,72 +24,72 @@
     "mobile"
   ],
   "dependencies": {
-    "bplist-parser": "0.0.6",
-    "broccoli": "0.16.2",
+    "bplist-parser": "0.1.0",
+    "broccoli": "0.16.4",
     "broccoli-funnel": "0.2.3",
     "bufferpack": "0.0.6",
     "bufferutil": "https://github.com/telerik/bufferutil/tarball/master",
-    "byline": "4.1.1",
-    "chalk": "1.0.0",
+    "byline": "4.2.1",
+    "chalk": "1.1.0",
     "cli-table": "https://github.com/telerik/cli-table/tarball/master",
```

Comment:
we should not target master, can you update this dependency as well, the target should be: 
`"https://github.com/telerik/cli-table/tarball/v0.3.1.1",`